compio-quic/examples/client.rs

```diff
@@ -0,0 +1,41 @@
+use std::net::{IpAddr, Ipv6Addr, SocketAddr};
```

Comment:
Name the examples in this crate as quic-*, e.g., `quic-client`, `quic-dispatcher`, `quic-server`.